mail_tracking_mailgun/models/ir_config_parameter.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2023 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class IrConfigParameter(models.Model):
+    _inherit = "ir.config_parameter"
+
+    @api.model
+    def _get_mailgun_timeout(self):
+        """Avoid waiting forever in requests"""
+        return self.get_param("mailgun.timeout", 10)
```

Comment:
Why putting this method in this model? And without sudo? I'm not finding too much advantage in it.

Reply:
Yeah, it isn't ideal. The thing was that having the default timeout spreaded around the code didn't seem very practical in case we wanted to adjust that value in the future.

For the sudo I'm respecting the default access restrictions so none can get that value without permission.

Reply:
But previous calls have sudo everywhere and now there isn't such sudo, and if you want to save, do it with sudo, but the formula I prefer is to put a global constant MAILGUN_TIMEOUT and import it.

Reply:
Ok, I've done it that way